index.html

```diff
@@ -0,0 +1,169 @@
+
+            <option value="2">Projects</option>
+
+            <option value="3">Community</option>
+
+            <option value="4">Open Source</option>
+          </select>
+
+          <div class="form-main">
+            <div class="form-main">
+              <p>What would you like to see improved? (Check all that apply)</p>
+            </div>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="1" id="checkbox1" />
+              <label for="checkbox1">Front-end Projects</label>
+            </div>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="2" id="checkbox2" />
+              <label for="checkbox2">Back-end Projects</label>
+            </div>
+
+            <input type="checkbox" name="checkbox" value="3" id="checkbox3" />
+            <label for="checkbox3">Data Visualization</label>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="4" id="checkbox4" />
+              <label for="checkbox4">Challenges</label>
+            </div>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="5" id="checkbox5" />
+              <label for="checkbox5">Open Source Community</label>
+            </div>
+
+            <input type="checkbox" name="checkbox" value="6" id="checkbox6" />
+            <label for="checkbox6">Gitter help rooms</label>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="7" id="checkbox7" />
+              <label for="checkbox7">Videos</label>
+            </div>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="8" id="checkbox8" />
+              <label for="checkbox8">City Meetups</label>
+            </div>
+
+            <div class="form-main">
+              <input type="checkbox" name="checkbox" value="9" id="checkbox9" />
+              <label for="checkbox9">Wiki</label>
+            </div>
+
+            <div class="form-main">
+              <input
+                type="checkbox"
+                name="checkbox"
+                value="10"
+                id="checkbox10"
+              />
+              <label for="checkbox10">Forum</label>
+            </div>
+
+            <div class="form-main">
+              <input
+                type="checkbox"
+                name="checkbox"
+                value="11"
+                id="checkbox11"
+              />
+              <label for="checkbox11">Additional Courses</label>
+            </div>
+          </div>
+
+          <div class="form-main">
+            <p>Any comments or suggestions?</p>
+            <textarea name="" id="" cols="30" rows="10"></textarea>
+          </div>
+
+          <div class="submit-button">
+            <button type="submit" id="submit">Submit</button>
+          </div>
+        </form>
+      </div>
+    </div>
```

Comment:
1. I think it would be better to use appropriate HTML5 semantic tags throughout the code.
2. Please I suggest you make use of the class attribute more often than using the id attribute